frameworks/digital-outcomes-and-specialists-2/questions/services/businessAnalystLocations.yml

```diff
@@ -1,4 +1,5 @@
 question: Where will your business analyst work?
+question-advice: Choose all the locations where the specialist can work at buyers’ sites, and whether they can work offsite.
```

Comment:
This should have an underscore rather than a dash (which is why the tests are failing), i.e. `question_advice` rather than `question-advice`.